test/README.md

```diff
@@ -1 +1,12 @@
-See https://github.com/WebAssembly/wasi-testsuite
+# Running tests
+The following command executes wasi-threads tests using wasmtime runtime.
+
+```bash
+git clone -b prod/testsuite-all https://github.com/WebAssembly/wasi-testsuite
+cd wasi-testsuite/
+TEST_RUNTIME_EXE="wasmtime <TBD flags>" python3 test-runner/wasi_test_runner.py \
```

Comment:
@abrown would you be able to help with wasmtime flags to enable threads?

Reply:
I think this needs `--wasm-features=threads --wasi-modules=experimental-wasi-threads`. Now, it might be good to note (or not, it's up to you) that these are not yet merged into the `main` branch (waiting on https://github.com/bytecodealliance/wasmtime/pull/5484 to merge...).